source/Ocl/Namers/IOclNamer.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Octopus.Ocl.Namers
+{
+    public interface IOclNamer
```

Comment:
If this interface is also going to be responsible for naming/matching enum values, we might need to tweak the enum PR a bit before it gets merged.

Reply:
I don't think it will be used for enums. This is to name the left hand side of the attribute (or the name of the block). Enum values should always be in a quoted string.